jobs/release.Jenkinsfile

```diff
@@ -239,19 +240,19 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
                 parallel push_containers.collectEntries{configname, val -> [configname, {
                     withCredentials([file(variable: 'REGISTRY_SECRET',
                                           credentialsId: 'oscontainer-push-registry-secret')]) {
-                        def repo = registry_repos[configname]
-                        def (artifact, metajsonname, tag_suffix) = val
-                        if (pipecfg.hotfix) {
-                            // this is a hotfix build; include the hotfix name
-                            // in the tag suffix so we don't clobber official
-                            // tags
-                            tag_suffix += "-hotfix-${pipecfg.hotfix.name}"
+                        def repo = registry_repos[configname]['registry']
+                        def (artifact, metajsonname) = val
+                        def tag_args = []
+                        for (tag in registry_repos[configname]['tags']) {
+                            if (pipecfg.hotfix) {
+                                // this is a hotfix build; include the hotfix name
+                                // in the tag suffix so we don't clobber official
+                                // tags
+                                tag += "-hotfix-${pipecfg.hotfix.name}"
+                            }
```

Comment:
Seems like it'd be cleaner to have this be part of `get_registry_repos` now that it does tag stuff?

Reply:
And then here we'd have:

```groovy
def tag_args = registry_repos[configname].tags.collect{"--tag=$it"}
```